test/python_client_test.py

```diff
@@ -103,43 +104,61 @@ def test_success_get_srv_keyspace(self):
     with self.assertRaises(ValueError):
       small.keyspace_id_to_shard_name_for_db_type(0x6000000000000000, 'replica')
 
+  KEYSPACE_ID_80 = struct.Struct('!Q').pack(0x80)
```

Comment:
Alternatively, you could use `struct.Struct('Q').pack(0x80)`, but I guess that might be even more confusing.